text/0060-tasks-api.md

```diff
@@ -132,6 +133,11 @@ Since the creation of a dump is not a task associated with a particular index, i
 
 Fully qualified and summarized task objects related to a dump creation display a `null` `indexUid` field.
 
+##### snapshotCreation
+N/A
+
+The `details` object is specified to `null` for a `snapshotCreation` task.
```

Comment:
I am pretty sure that makes sense, but I am like: _can't we find anything interesting to put in there_...

Reply:
Same! Let's go with that!